src/components/Item.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { View, Image, Text, ImageProps, StyleSheet } from "react-native";
+import Octicons from "@expo/vector-icons/Octicons";
+import { Typography } from "@/theme/typography";
+import { ColorTheme } from "@/theme/colors";
+
+export const Item = (props: ItemProps) => {
+  const { heading, subtitle, body, headingSize, ...avatarProps } = props;
+  const bodyProps = { heading, subtitle, body, headingSize };
+  return (
+    <View style={ItemStyles.row}>
+      <ItemAvatar {...avatarProps} />
+      <ItemBody {...bodyProps} />
+    </View>
+  );
+};
+
+export const ItemStack = (props: ItemStackProps) => {
+  const { images, ...bodyProps } = props;
+  return (
+    <View>
+      <View style={ItemStyles.row}>
+        {images.map((imageProps, index) => (
+          <ItemAvatar key={index} {...imageProps} />
+        ))}
+      </View>
+      <ItemBody {...bodyProps} />
+    </View>
+  );
+};
+
+export const ItemAvatar = (props) => {
+  const { source, iconName, badgeName } = props;
```

Comment:
Not a huge fan of doing this on the first line of every function. You could destructure in the params and avoid the clutter